.github/workflows/gradle.version.yaml

```diff
@@ -14,21 +36,63 @@ jobs:
   compute-version:
     runs-on: ubuntu-latest
     outputs:
-      version: ${{ steps.version.outputs.version }}
+      minecraft_version: ${{ steps.generateMinecraftVersion.outputs.minecraft_version }}
+      mod_version: ${{ steps.generateModVersion.outputs.version }}
+      full_version: ${{ steps.generateFullVersion.outputs.full_version }}
+      tag_version: ${{ steps.generateTagVersion.outputs.tag_version }}
+      tag_regex: ${{ steps.generateTagRegex.outputs.tag_regex }}
+      release_text: ${{ steps.generateReleaseText.outputs.release_text }}
     steps:
       - id: checkout
         name: "📦 Checkout"
         uses: actions/checkout@v4
         with:
-          fetch-depth: 0
+          fetch-depth: 1000
```

Comment:
At Orion: i think we can lower this something more sensible like 100? (not sure where 1k comes from, but iirc neo has changelog generation since fixed version so that's why)